github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-periodics.yaml

```diff
@@ -211,7 +211,39 @@ periodics:
           - "/app/robots/cmd/indexpagecreator/app.binary"
         args:
           - --dry-run=false
-
+- name: periodic-publish-multiarch-bootstrap-image
```

Comment:
wondering if there is an easy way to skip building in case since the last _successful_ build of it there were no changes
(anyhow not in the scope of this PR)